From 08ffa5532248920c1cd418834e6f5f81129ecc11 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Tue, 7 Jan 2014 14:58:59 +0800
Subject: [PATCH 283/310] ENGR00293648 Settings: Can't activate device
 administrator in Settings

Align the feature list in frameworks/base/data/etc/tablet_core_hardware.xml
Add android.software.device_admin feature for all i.MX platform

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/required_hardware.xml       | 2 ++
 arm2_6sl/required_hardware.xml       | 2 ++
 evk_6sl/required_hardware.xml        | 2 ++
 hdmidongle_6dq/required_hardware.xml | 2 ++
 imx53_smd/required_hardware.xml      | 2 ++
 sabreauto_6q/required_hardware.xml   | 2 ++
 sabresd_6dq/required_hardware.xml    | 2 ++
 7 files changed, 14 insertions(+)

diff --git a/arm2_6dq/required_hardware.xml b/arm2_6dq/required_hardware.xml
index 83964bc..c4e0558 100644
--- a/arm2_6dq/required_hardware.xml
+++ b/arm2_6dq/required_hardware.xml
@@ -41,4 +41,6 @@
     <feature name="android.software.app_widgets" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
+    <!-- Feature to specify if the device supports adding device admins. -->
+    <feature name="android.software.device_admin" />
 </permissions>
diff --git a/arm2_6sl/required_hardware.xml b/arm2_6sl/required_hardware.xml
index 83964bc..c4e0558 100644
--- a/arm2_6sl/required_hardware.xml
+++ b/arm2_6sl/required_hardware.xml
@@ -41,4 +41,6 @@
     <feature name="android.software.app_widgets" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
+    <!-- Feature to specify if the device supports adding device admins. -->
+    <feature name="android.software.device_admin" />
 </permissions>
diff --git a/evk_6sl/required_hardware.xml b/evk_6sl/required_hardware.xml
index 5513a1e..6bb6dbe 100644
--- a/evk_6sl/required_hardware.xml
+++ b/evk_6sl/required_hardware.xml
@@ -31,6 +31,8 @@
     <feature name="android.software.app_widgets" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
+    <!-- Feature to specify if the device supports adding device admins. -->
+    <feature name="android.software.device_admin" />
 
     <!-- This is added by FSL. -->
     <feature name="android.software.live_wallpaper" />
diff --git a/hdmidongle_6dq/required_hardware.xml b/hdmidongle_6dq/required_hardware.xml
index 3cac43c..b0c33d5 100644
--- a/hdmidongle_6dq/required_hardware.xml
+++ b/hdmidongle_6dq/required_hardware.xml
@@ -32,6 +32,8 @@
     <feature name="android.software.app_widgets" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
+    <!-- Feature to specify if the device supports adding device admins. -->
+    <feature name="android.software.device_admin" />
 
     <!-- This is added by FSL. -->
     <feature name="android.software.live_wallpaper" />
diff --git a/imx53_smd/required_hardware.xml b/imx53_smd/required_hardware.xml
index 10f18d1..26b7797 100644
--- a/imx53_smd/required_hardware.xml
+++ b/imx53_smd/required_hardware.xml
@@ -41,4 +41,6 @@
     <feature name="android.software.app_widgets" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
+    <!-- Feature to specify if the device supports adding device admins. -->
+    <feature name="android.software.device_admin" />
 </permissions>
diff --git a/sabreauto_6q/required_hardware.xml b/sabreauto_6q/required_hardware.xml
index ec8a5a1..21d809d 100644
--- a/sabreauto_6q/required_hardware.xml
+++ b/sabreauto_6q/required_hardware.xml
@@ -35,6 +35,8 @@
     <feature name="android.software.app_widgets" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
+    <!-- Feature to specify if the device supports adding device admins. -->
+    <feature name="android.software.device_admin" />
 
     <!-- This is added by FSL. -->
     <feature name="android.software.live_wallpaper" />
diff --git a/sabresd_6dq/required_hardware.xml b/sabresd_6dq/required_hardware.xml
index 06024be..1c1dbc2 100644
--- a/sabresd_6dq/required_hardware.xml
+++ b/sabresd_6dq/required_hardware.xml
@@ -36,6 +36,8 @@
     <feature name="android.software.app_widgets" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
+    <!-- Feature to specify if the device supports adding device admins. -->
+    <feature name="android.software.device_admin" />
 
     <!-- This is added by FSL. -->
     <feature name="android.software.live_wallpaper" />
-- 
1.8.0

